README.md

```diff
@@ -517,13 +517,214 @@ Escrever testes de integração restritos para um serviço separado é bastante
+* Implementar o serviço fornecedor de acordo com o contrato definido
+* Jogar a especificação de interface para a equipe de consumo
+* Aguardar até que eles implementem a parte deles do consumo da interface
+* Executar alguns testes manuais em larga escala do sistema para ver se tudo funciona
+* Esperar que ambas as equipes sigam a definição da interface para sempre e não estraguem tudo
+
+As equipes de desenvolvimento de software mais modernas substituíram as etapas 5 e 6 por algo mais automatizado: Testes de contrato automatizados garantem que as implementações no lado do consumidor e do fornecedor ainda sigam o contrato definido. Eles funcionam como uma boa suíte de teste de regressão e garantem que as variações do contrato serão notadas precocemente.
+
+Em uma organização mais ágil, você deve seguir a rota mais eficiente e menos desperdiçadora. Você desenvolve suas aplicações dentro da mesma organização. Realmente não deve ser muito difícil falar diretamente com os desenvolvedores de outros serviços em vez de lançar documentação excessivamente detalhada por cima do muro. Afinal, eles são seus colegas de trabalho e não um fornecedor terceirizado com o qual você só pode falar via suporte ao cliente ou contratos à prova de balas.
+
+Os testes de Contrato Impulsionados pelo Consumidor (CDC - Consumer-Driven Contract tests) permitem que os consumidores conduzam a implementação de um contrato. Usando CDC, os consumidores de uma interface escrevem testes que verificam a interface para todos os dados que eles precisam dessa interface. A equipe consumidora, então, publica esses testes para que a equipe provedora possa buscar e executar esses testes facilmente. A equipe fornecedora agora pode desenvolver sua API executando os testes CDC. Depois que todos os testes passam, eles sabem que implementaram tudo o que a equipe consumidora precisa.
+
+
+![Testes de contrato](/assets/cdc_tests.png "Especificação de testes de contrato")
+**Figura 9:** Os testes de contrato garantem que o provedor e todos os consumidores de uma interface sigam o contrato de interface definido. Com os testes CDC, os consumidores de uma interface publicam seus requisitos na forma de testes automatizados; os provedores buscam e executam esses testes continuamente.
+
+Esta abordagem permite que a equipe provedora implemente apenas o que é realmente necessário (mantendo as coisas simples, YAGNI e tudo mais). A equipe que fornece a interface deve buscar e executar continuamente esses testes CDC (em seu pipeline de build) para detectar quaisquer mudanças quebradoras imediatamente. Se eles quebrarem a interface, seus testes CDC falharão, impedindo que as mudanças quebradoras entrem em produção. Enquanto os testes permanecerem verdes, a equipe pode fazer quaisquer mudanças que desejar sem se preocupar com outras equipes. A abordagem de Contrato Impulsionado pelo Consumidor deixaria você com um processo parecido com este:
```

Comment:
pipeline de construção, com a palavra pipeline em itálico.
Pensar em outra tradução para "mudanças quebradoras". Achei a palavra estranha. Talvez, "mudanças que "quebram" o contrato".

Reply:
Alteração realizada.